### PR TITLE
Qt5.4.1

### PR DESCRIPTION
**Pull request by luzpaz**
Updated URLs

### CODE REVIEW

File: Library/Formula/qt5.rb [hunk 1]
```diff
@@ -11,15 +11,15 @@ def message; <<-EOS.undent
...
-  end
+  url "https://download.qt.io/official_releases/qt/5.4/5.4.1/single/qt-everywhere-opensource-src-5.4.1.tar.xz"
+  mirror "http://qtmirror.ics.com/pub/qtproject/official_releases/qt/5.4/5.4.1/single/qt-everywhere-opensource-src-5.4.1.tar.xz"
+  sha1 "daa3373af8d6656a1066ff23bc9100b5ca004ecf"
+
+bottle do
+  sha1 "072ed2c806664fd1da3ba7c90c8e4887509fb91b" => :yosemite
+  sha1 "1ca730d96a962a5c4fcbd605542b7bfb528d6c58" => :mavericks
+  sha1 "a6bbd39629a69c35c8a5d5e8ede4b6c752e3aecf" => :mountain_lion
+end
```
Comment (mikemcquaid) [inline Library/Formula/qt5.rb]: Don't change the indentation here, thanks.

**Comment by luzpaz** [comment]
Brewbot seems to be down. When it's functioning..can someone retrigger this? Thanks!

**Comment by luzpaz** [comment]
Encountering same issues as per https://github.com/Homebrew/homebrew/pull/34929#issuecomment-66898185

**Comment by mikemcquaid** [comment]
@luzpaz We can ignore that for now. The bigger issue is this is taking too long for the bot. I'll need to wait until a quiet time and then kick it off again.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `qt5 5.4.1`.

**Pull request closed without merging**